src/invoices/Voucher/VoucherEditForm/VoucherEditContainer.js

```diff
@@ -0,0 +1,122 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { get } from 'lodash';
+
+import {
+  Paneset,
+  LoadingView,
+} from '@folio/stripes/components';
+import {
+  stripesConnect,
+} from '@folio/stripes/core';
+import {
+  getConfigSetting,
+  useShowCallout,
+} from '@folio/stripes-acq-components';
+
+import {
+  invoiceResource,
+  VOUCHER_BY_ID,
+  configVoucherNumber,
+} from '../../../common/resources';
+import VoucherEditForm from './VoucherEditForm';
+
+const VoucherEditContainer = ({
+  match: { params },
+  mutator,
+  resources,
+  history,
+  location,
+}) => {
+  const [voucher, setVoucher] = useState();
+  const [vendorInvoiceNo, setVoucherInvoiceNo] = useState();
+  const [isAllowVoucherNumberEdit, setAllowVoucherNumberEdit] = useState();
+  const showCallout = useShowCallout();
+
+  useEffect(() => {
+    mutator.voucher.GET({
+      params: {
+        id: params.voucherId,
+      },
+    })
+      .then(setVoucher)
+      .catch(() => {
+        showCallout({ messageId: 'ui-invoice.errors.voucherLoadingFailed', type: 'error' });
+      });
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    const invoiceNo = get(resources, ['invoice', 'records', 0, 'vendorInvoiceNo']);
+    const { allowVoucherNumberEdit } = getConfigSetting(get(resources, 'configVoucherNumber.records', {}));
+
+    setAllowVoucherNumberEdit(allowVoucherNumberEdit);
+    setVoucherInvoiceNo(invoiceNo);
+  }, [resources]);

```

Comment:
do you really need `resources` here? do you know how this property is changed? my previous comment was about manual loading to remove resources property